Site1.Master

```diff
@@ -55,39 +55,40 @@
 
                    <ul class="nav justify-content-end">
                         <li class="nav-item active">
-                            <asp:LinkButton class="nav-link" ID="LinkButton4" runat="server" OnClick="LinkButton4_Click">View Task</asp:LinkButton>
+                            <asp:LinkButton class="nav-link" ID="ViewTask" runat="server" OnClick="BttnViewTask_Click">View Task</asp:LinkButton>
```

Comment:
### Wrong naming convention, again. 

Your naming convention needs to correlate with your element

lbViewTask instead of ViewTask.